src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs

```diff
@@ -301,7 +353,7 @@ private bool TryResolveSdkUsingSpecifiedResolvers(IList<SdkResolver> resolvers,
 
                 if (result.Success)
                 {
-                    LogWarnings(loggingContext, sdkReferenceLocation, result);
+                    LogWarnings(loggingContext, sdkReferenceLocation, result.Warnings);
```

Comment:
Resolver could fail silently without any message at all, returning null (see "continue" couple of lines above this line). No error or warning or log messages are added there. Therefore, it is nowhere in logs, and I feel that is not good. 
What do you think about adding some default message for this case to be shown as well, with the meaning "Resolver <ResolverName> was run but returned null.". Either log, or warning, or error. Error might be fine, I think, in case of failed resolution there will be errors for every resolver that was called.

Reply:
Pushed a proposed change for this 🙂 Let me know if you don't like the wording.